### ISSUE TITLE
CocoaPods master repo missing Parse-iOS-SDK

### ISSUE THREAD

**Issue by bdbergeron** [twobitlabs/AnalyticsKit#10]
I was hoping that by the time my addition of Parse to AnalyticsKit was merged in that my Parse SDKs would be merged into the master CocoaPods repo. Unfortunately, at this time, it is not. For a temporary fix, just execute  `repo add bdbergeron https://github.com/bdbergeron/Specs` in your terminal to add my fork of the CocoaPods specs. This needs to be done before executing `pod install`.
